master/internal/db/postgres_snapshots_test.go

```diff
@@ -17,19 +17,20 @@ import (
 )
 
 func TestCustomSearcherSnapshot(t *testing.T) {
-	etc.SetRootPath(RootFromDB)
+	err := etc.SetRootPath(RootFromDB)
+	require.NoError(t, err)
 	db := MustResolveTestPostgres(t)
 	MustMigrateTestPostgres(t, db, MigrationsFromDB)
 	user := RequireMockUser(t, db)
 	exp := RequireMockExperiment(t, db, user)
 
-	config := expconf.SearcherConfig{
-		RawCustomConfig: &expconf.CustomConfig{},
+	config := expconf.SearcherConfig{ //nolint
```

Comment:
It was complaining that other Configs (RawSingleConfig) are not provided. But we shouldn't have to provide that.

Reply:
You should be able to make these `//nolint:exhaustivestruct` to make explicit what is being ignored.